src/pendulum/utils/_zoneinfo.py

```diff
@@ -1,74 +1,12 @@
 from __future__ import annotations
 
-import sys
+from zoneinfo import TZPATH
+from zoneinfo import InvalidTZPathWarning
+from zoneinfo import ZoneInfo
+from zoneinfo import ZoneInfoNotFoundError
+from zoneinfo import available_timezones
+from zoneinfo import reset_tzpath
```

Comment:
This shouldn't be needed anymore, all uses of `pendulum.utils._zoneinfo` should be replaced with direct usage of zoneinfo and this module should be removed.

Reply:
Done!
Thanks for triggering codspeed!
Regarding the performances regression, I am not sure if this is acceptable.
There is a loss of "performance" according to codspeed on [v2.0.0 release](https://github.com/CodSpeedHQ/pytest-codspeed/releases/tag/v2.0.0).